### PR TITLE
chore: [IOPID-1720] Add features to OperationResultScreenContent 

### PR DESCRIPTION
**Pull request by Ladirico**
## Short description
Generalization of the component to make it usable in more cases and retro-compatibility of use

## List of changes proposed in this pull request
- Add a component, `ComposedBodyFromArray`, that receives as input an array containing text and styles and maps them and automatically creates the text from multiple separate strings
- Integrate examples in DS route

## Examples
<p>

| Subtitle as string | Subtitle as array | 
| - | - |
| Without buttonType props and subtitle as string | Without buttonType props and subtitle as array | 

</p>

### CODE REVIEW

File: ts/components/screens/OperationResultScreenContent.tsx [hunk 1]
```diff
@@ -58,24 +111,45 @@ const OperationResultScreenContent = ({
...
+        )}
+      {secondaryAction && secondaryAction.buttonType === "ButtonSolid" && (
         <View style={IOStyles.alignCenter}>
           <VSpacer size={24} />
           <View>
-            <ButtonLink {...secondaryAction} />
+            <ButtonSolid {...(secondaryAction as ButtonSolidProps)} />
           </View>
         </View>
       )}
```
Comment (shadowsheep1) [inline ts/components/screens/OperationResultScreenContent.tsx:155]: What about this?

```ts
      {action && (
        <View style={IOStyles.alignCenter}>
          <VSpacer size={24} />
          <View>
            {action.buttonType === "ButtonLink" ? (
              <ButtonLink {...action} />
            ) : (
              <ButtonSolid {...(action as ButtonSolidProps)} />
            )}
          </View>
        </View>
      )}
      {secondaryAction && (
        <View style={IOStyles.alignCenter}>
          <VSpacer size={24} />
          <View>
            {secondaryAction.buttonType === "ButtonSolid" ? (
              <ButtonSolid {...(secondaryAction as ButtonSolidProps)} />
            ) : (
              <ButtonLink {...secondaryAction} />
            )}
          </View>
        </View>
      )}
```

What do you think?

**Review by shadowsheep1 — CHANGES_REQUESTED** [review]

File: ts/components/screens/OperationResultScreenContent.tsx (hunk 1 above)
Comment (Ladirico) [reply ts/components/screens/OperationResultScreenContent.tsx]: as a consequence of changes requested by the design team, I deleted this part of the code.

Comment (shadowsheep1) [reply ts/components/screens/OperationResultScreenContent.tsx]: Do you mean that action is always a solid button and secondAction is always a link button? Am I right?

**Review by shadowsheep1 — APPROVED** [review]
LGTM

**Pull request merged**